src/hivpy/cli.py

```diff
@@ -0,0 +1,38 @@
+import configparser
+import argparse
+import pathlib
+from .experiment import create_experiment, create_output, run_experiment
+
+'''
+Maybe we may want to register parameters/properties in future?
+'''
+def register_parameters():
+    parser = argparse.ArgumentParser(description="register model parameters")
+    parser.add_argument("parameters", type=pathlib.Path, help="register_parameters parameters.csv")
+    args = parser.parse_args()
+    parameter_filepath = args.parameters
+
+'''
+Running a HIV model simulation
+Assuming there is a hivpy.conf file the command is
+$run_model hivpy.conf
+'''
+def run_model():
+    parser = argparse.ArgumentParser(description="run a simulation")
+    parser.add_argument("input", type=pathlib.Path, help="run_model -i config.conf")
```

Comment:
The help string has an extra `-i`, but also I think it would be better to describe the particular argument (users will get the suggested usage string automatically if you use the wrong syntax).

Reply:
Removed the -i. I will see how the help in the command line can provide more info on expected arguments

Reply:
I think it's done by default if the user doesn't give the right arguments: the expected usage is automatically assembled and displayed. I don't think you need to do anything extra.

The `help` string is meant to describe each input argument (like "a file with the parameters of the simulation") so it can be displayed in the overall help/usage message.